server/data/reference1/users.txt

```diff
@@ -0,0 +1 @@
+Demo:pass
```

Comment:
want to save this in export too, to present to user after initialisation import, so that they know what users are available